### PR TITLE
[Phase 1] LPS-174289 Remove Stream and Optional usages for module: portal-search-test-util part 1

### PR DESCRIPTION
**Pull request by julschong**
resubmitted from #3593

### CODE REVIEW

File: modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/FieldValuesAssert.java [hunk 1]
```diff
@@ -160,20 +158,21 @@ private static void _addFields(
...
+		Map<String, String> map1, Predicate<String> predicate) {
 
 		if (predicate == null) {
-			return map;
+			return map1;
 		}
 
-		Stream<Map.Entry<String, String>> stream = SearchStreamUtil.stream(
-			map.entrySet());
+		Map<String, String> map2 = new HashMap<>();
```
Comment (julschong) [inline modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/FieldValuesAssert.java:167]: changed here

File: modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/FacetsAssert.java [hunk 2]
```diff
@@ -43,15 +42,11 @@ public static void assertFrequencies(
...
 		Assert.assertEquals(
 			message, expected,
-			stream.map(
-				FacetsAssert::toString
-			).collect(
-				Collectors.toList()
-			).toString());
+			String.valueOf(
+				TransformUtil.transform(
+					termCollectors, FacetsAssert::toString)));
```
Comment (julschong) [inline modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/FacetsAssert.java:49]: changed here by using String.valueOf

**Comment by Kvale1733** [comment]
ci:test:search

**Comment by Kvale1733** [comment]
@julschong No regressions.

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/130685

@julschong Please see my change. I noticed that some of the things I commented were already picked by Tina in https://github.com/julschong/liferay-portal/pull/760, please pay attention to the PRs sent back to you, thanks!

**Pull request closed without merging**